### PR TITLE
[JBPM-10089] Orphan sessions in memory due to an exception on PerRequestRuntimeManager

### PR DESCRIPTION
**Pull request by elguardian**
Jira: https://issues.redhat.com/browse/JBPM-10089

### CODE REVIEW

**Comment by gmunozfe** [comment]
jenkins retest this

File: jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/PerRequestRuntimeManager.java [hunk 1]
```diff
@@ -167,13 +169,25 @@ public void disposeRuntimeEngine(RuntimeEngine runtime) {
...
     	} catch (Exception e) {
     	    local.get().remove(identifier);
     	    throw new RuntimeException(e);
     	}
     }
 
+    @Override
+    protected Environment getEnvironment(RuntimeEngine runtimeEngine) {
+        if (runtimeEngine instanceof RuntimeEngineImpl) {
+            if (((RuntimeEngineImpl) runtimeEngine).isInitialized()) {
```
Comment (gmunozfe) [inline jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/PerRequestRuntimeManager.java:184]: both "ifs" could be merged into one to enhance readibility

**Review by gmunozfe — COMMENTED** [review]
Could you add a test for completeness?

**Comment by elguardian** [comment]
Jenkins retest this please

**Pull request merged**